src/components/board/board.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import "../../styles/styles.css";
+import Square from "../square/square";
+
+
+const files = ["a", "b", "c", "d", "e", "f", "g", "h"];
```

Comment:
Just a nit pick thing from my old age - constants are usually all caps - for me anyway. Not a major change but helps to know further in the code what is a constant value and what isn't.

`const FILES = ["a", "b", "c", "d", "e", "f", "g", "h"];`

See below for possibly capitalizing these.